doc/modules/linear_model.rst

```diff
@@ -39,14 +39,16 @@ solves a problem of the form:
 
 :class:`LinearRegression` will take in its ``fit`` method arrays X, y
 and will store the coefficients :math:`w` of the linear model in its
-``coef_`` member::
+``coef_`` and ``intercept_`` members::
 
     >>> from sklearn import linear_model
     >>> reg = linear_model.LinearRegression()
-    >>> reg.fit([[0, 0], [1, 1], [2, 2]], [0, 1, 2])
+    >>> reg.fit([[0, 0], [1, 1], [2, 2]], [0, 1, 3])
```

Comment:
Following @thomasjpfan's suggestion, this should be changed like so
```suggestion
    >>> reg.fit([[0, 0], [1, 1], [2, 2]], [1, 2, 3])
```

Your change was `[0,1,2]->[0,1,3]` instead of `[0,1,2]->[1,2,3]` hence the doc-test failures since that changes the actual weights, not just shifting the intercept.

Reply:
Corrected, I am sorry for that.